urdf2webots/writeProto.py

```diff
@@ -501,8 +501,8 @@ def URDFShape(proto, link, level, normal=False):
             if name is None:
                 if visualNode.geometry.name is not None:
                     name = computeDefName(visualNode.geometry.name)
+            name = robotNameMain + '_' + name if robotNameMain else name
```

Comment:
This is an essential fix, that I already included in the batch conversion PR, It makes sure, that the proto names are correct.